features/welcome/actions.js

```diff
@@ -97,13 +116,29 @@ export function conferenceJoined(conference) {
 /**
  * Create an action for when the signaling connection has been lost.
  *
- * @param {string} message - Error message.
+ * @param {string} [message] - Error message.
  * @returns {{type: JITSI_CLIENT_DISCONNECTED, message: string}}
  */
 export function connectionDisconnected(message) {
-    return {
-        type: JITSI_CLIENT_DISCONNECTED,
-        message
+    return dispatch => {
+        dispatch(removeLocalParticipant());
+
+        return dispatch(removeLocalTracks())
+            .then(() => dispatch({ type: JITSI_CLIENT_DISCONNECTED, message }));
+    };
+}
+
+/**
+ * Create an action for when the signaling conference was left.
+ *
+ * @returns {{type: JITSI_CONFERENCE_LEFT, message: string}}
+ */
+export function conferenceLeft() {
+    return dispatch => {
+        dispatch(resetToolbar());
```

Comment:
In my next PR #40  I'm refactoring 'toolbar' feature. Most of its code is moved into 'features/base/media', including 'resetMedia' action - https://github.com/jitsi/jitsi-meet-react/pull/40/files#diff-942678cee57360d64010c8e87de3cec6R329.

So then __leave()__ action is using only "base" actions inside - https://github.com/jitsi/jitsi-meet-react/pull/40/files#diff-50716ee250158cbd9ef72f1ab15d7ec3R196

I can suggest to leave it in this PR as is and wait until #40 is merged. Or we can close this PR and concentrate on reviewing and merging #40